### ISSUE TITLE
Documentation on Flashing Firmware Directly

### ISSUE THREAD

**Issue by lukeinator42** [moothyknight/HEG_ESP32_Delobotomizer#3]
Hi,

Is there a way to flash firmware directly to the ESP32? If so it might be good to add documentation on how to do that. I recently received my headband, but it didn't create a Wi-Fi network to connect to, and the device also won't connect over Bluetooth or USB. I wonder if it didn't get its firmware flashed? 

I also tried flashing the firmware with esptool but it still won't connect or create the wifi network.

Thanks,
Lukas

**Comment by joshbrew** [comment]
Really? Nothing? All the ones I shipped out are triple checked to try to avoid this. They should all just be ready to use out of the box. Otherwise there should be instructions for working in Arduino IDE as that's what I use.

**Comment by lukeinator42** [comment]
Yeah, nothing. I also cloned the repo, copied the libraries from the sketch folder to my arduino ide libraries and tried compiling the firmware but I'm getting compiling errors. What version of the ESP32 board is used in the headband? I might've selected the wrong one (I think I selected Lolin D32).
